index.js

```diff
@@ -201,7 +220,60 @@ walletRouter.post('/send-tx', (req, res) => {
   const wallet = req.wallet;
   const outputs = req.body.outputs;
   const inputs = req.body.inputs || [];
-  const ret = wallet.sendManyOutputsTransaction(outputs, inputs)
+  // Expects object with {'uid', 'name', 'symbol'}
+  const token = req.body.token || null;
+  const ret = wallet.sendManyOutputsTransaction(outputs, token, inputs)
+  if (ret.success) {
+    ret.promise.then((response) => {
+      res.send(response);
+    }, (error) => {
+      res.send({success: false, error});
+    });
+  } else {
+    res.send({success: false, error: ret.message});
+  }
+});
+
+walletRouter.post('/create-token', (req, res) => {
```

Comment:
Missing docstring.

Reply:
We don't really have any default docstring for APIs, should we start adding docstring to each endpoint?
What we have here is the `api-docs.js` which was not updated and I've just added all new endpoints and updated old ones. What do you think? Should we have the api-docs file and the docstring? Or a simple docstring version?

Reply:
You're right. I think the `api-docs.js` is enough. Maybe we can add a docstring like "For the docs, see api-docs.js". What do you think?

Reply:
I've added a short description of each endpoint and the reference to the docs